python-modules-list.sh

```diff
@@ -1,8 +1,12 @@
 package: Python-modules-list
 version: "1.0"
 env:
-  PIP_REQUIREMENTS: |
+  PIP_BASE_REQUIREMENTS: |
+    pip==23.0.1
+    setuptools==65.5.1
+    wheel
```

Comment:
Can you specify a version for wheel here? Otherwise aliBuild won't rebuild Python-modules when wheel is updated upstream.

Reply:
done